packages/compartment-mapper/test/test-bundle.js

```diff
@@ -77,3 +77,49 @@ test('equivalent archive behaves the same as bundle', async t => {
   });
   t.deepEqual(log, expectedLog);
 });
+
+// This is failing because it requires support for missing dependencies.
+// Cannot bundle: encountered deferredError Cannot find file for internal module "./spam"
+test.failing('bundle cjs-compat', async t => {
+  const cjsFixture = new URL(
+    'fixtures-cjs-compat/node_modules/app/index.js',
+    import.meta.url,
+  ).toString();
+
+  const bundle = await makeBundle(read, cjsFixture);
+  const log = [];
+  const print = entry => {
+    log.push(entry);
+  };
+  const compartment = new Compartment({ print });
+  compartment.evaluate(bundle);
+  t.deepEqual(log, expectedLog);
+});
+
+test('bundle cjs-compat default-difficulties', async t => {
+  const cjsFixture = new URL(
+    'fixtures-cjs-compat/node_modules/default-difficulties/index.mjs',
+    import.meta.url,
+  ).toString();
+
+  const bundle = await makeBundle(read, cjsFixture);
+  fs.writeFileSync('bundle.js', bundle);
+  const compartment = new Compartment();
+  let namespace;
+  t.notThrows(() => {
+    namespace = compartment.evaluate(bundle);
+  });
```

Comment:
I suspect that awaiting here is sufficient since the intent is to stop the test if it throws. The subsequent errors will be less direct than failing here.

Reply:
`compartment.evaluate(bundle)` returns synchronously and does not return a promise, so i'm confused about the purpose of the `await`

Reply:
my bad, just call :-)